viewing_party/party.py

```diff
@@ -0,0 +1,132 @@
+#from Collections import Counter
+
+# WAVE 1 FUNCTIONS:
+
+# function to create new movie dictionary object
+def create_movie(movie_title, genre, rating):
+    if (movie_title and genre and rating):
+        return { "title":movie_title, "genre":genre, "rating":rating}
+    return None
+
+# function to add movie to 'watched' movie list
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+    
+# function to add movie to 'watchlist'
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+# function moves movies from 'watchlist' to 'watched'
+def watch_movie(user_data, movie_title):
+    for movie in user_data["watchlist"]:
+        if movie["title"] == movie_title:
+            # add movie to watched
+            user_data["watched"].append(movie)
+            #remove movie from watchlist
+            user_data["watchlist"].remove(movie)
+    return user_data
+
+
+
+# WAVE 2 FUNCTIONS:
+
+# function that returns average of 'watched' movie ratings
+def get_watched_avg_rating(user_data):
+    if user_data["watched"]:
+        my_ratings = [movie["rating"] for movie in user_data["watched"]]
+        avg_rating = sum(my_ratings) / len(my_ratings)
+        return avg_rating
+    return 0.0
+
+# QUESTION: AM I ALLOWED TO IMPORT COUNTER FROM COLLECTIONS?
```

Comment:
Yes.  I you don't need to `pip install` something it's okay.

Though if we haven't covered using the function in class it would be great if you could leave a comment explaining it so that we can be sure you understand it.  (And can clarify any misunderstanding that exists.)